pytest_fixtures/broker.py

```diff
@@ -9,36 +9,34 @@
 from robottelo.hosts import Satellite
 
 
+def satellite_factory(retry_limit=3, delay=300, **broker_args):
+    vmb = VMBroker(
+        host_classes={'host': Satellite}, workflow=settings.server.deploy_workflow, **broker_args
+    )
+    timeout = (1200 + delay) * retry_limit
+    sat = wait_for(
+        vmb.checkout, func_kwargs=broker_args, timeout=timeout, delay=delay, fail_condition=[]
+    )
+    return sat.out
+
+
+def capsule_factory(retry_limit=3, delay=300, **broker_args):
+    vmb = VMBroker(host_classes={'host': Satellite}, workflow='deploy-sat-capsule', **broker_args)
```

Comment:
This was addressed in another open PR, but pointing out my oversight here as well
```suggestion
    vmb = VMBroker(host_classes={'host': Capsule}, workflow='deploy-sat-capsule', **broker_args)
```

Reply:
Can you fix this in a specific PR, along with `deploy-sat-capsule` being read from config instead of hardcoded?

Reply:
This is now being resolved by #8514